pandas/tseries/index.py

```diff
@@ -1755,12 +1754,16 @@ def indexer_at_time(self, time, asof=False):
     def indexer_between_time(self, start_time, end_time, include_start=True,
                              include_end=True):
         """
-        Select values between particular times of day (e.g., 9:00-9:30AM)
+        Select values between particular times of day (e.g., 9:00-9:30AM).
+        
+        Return values of the index between two times.  If start_time or
+        end_time are strings then tseres.tools.to_time is used to convert to
+        a time object.
 
         Parameters
         ----------
-        start_time : datetime.time or string
-        end_time : datetime.time or string
+        start_time, end_time : datetime.time or string
+            Time or string in appropriate format (e.g., "%H:%M", "%I%M%p")
```

Comment:
``datetime.time`` or string in appropriate time-like format (and include the formats you have above)